src/main/webapp/app/quiz/participate/short-answer-question/short-answer-question.component.html

```diff
@@ -0,0 +1,78 @@
+<div class="sa-question" [ngClass]="{disabled: clickDisabled && !showResult, result: showResult && !forceSampleSolution, incorrect: (score || 0) < question.score && !forceSampleSolution}">
+    <h4 class="question-title-display"><span>{{questionIndex}})</span> {{question.title}}</h4>
+    <p *ngIf="questionText !== '' ">{{questionText}}</p>
+
+    <div *ngIf="!showResult && !clickDisabled">
+        <span [ngClass]="{'saQuestion-list': isList}" *ngFor="let textPart of textWithOutSpotsFirstParts; let i = index">
```

Comment:
rename textWithOutSpotsFirstParts --> I don't understand what you mean here

Reply:
Refactored in sa-quiz-statistic branch